src/PrestaShopBundle/Controller/Admin/Sell/CustomerService/OrderMessageController.php

```diff
@@ -0,0 +1,276 @@
+
+            if ($result->getIdentifiableObjectId()) {
+                $this->addFlash('success', $this->trans('Successful creation.', 'Admin.Notifications.Success'));
+
+                return $this->redirectToRoute('admin_order_messages_index');
+            }
+        } catch (Exception $e) {
+            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages()));
+        }
+
+        return $this->render('@PrestaShop/Admin/Sell/CustomerService/OrderMessage/create.html.twig', [
+            'help_link' => $this->generateSidebarLink($request->attributes->get('_legacy_controller')),
+            'enableSidebar' => true,
+            'layoutTitle' => $this->trans('Add new', 'Admin.Actions'),
+            'orderMessageForm' => $form->createView(),
+        ]);
+    }
+
+    /**
+     * Edit existing order message
+     *
+     * @AdminSecurity(
+     *     "is_granted(['update'], request.get('_legacy_controller'))",
+     *     redirectRoute="admin_order_messages_index"
+     * )
+     *
+     * @param int $orderMessageId
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function editAction(int $orderMessageId, Request $request): Response
+    {
+        $formBuilder = $this->get('prestashop.core.form.identifiable_object.builder.order_message_form_builder');
+        $formHandler = $this->get('prestashop.core.form.identifiable_object.handler.order_message_form_handler');
+
+        try {
+            /** @var EditableOrderMessage $editableOrderMessage */
+            $editableOrderMessage = $this->getQueryBus()->handle(new GetOrderMessageForEditing($orderMessageId));
+
+            $orderMessageName = $editableOrderMessage->getLocalizedName()[$this->getContextLangId()];
+
+            $form = $formBuilder->getFormFor($orderMessageId);
+            $form->handleRequest($request);
+
+            $result = $formHandler->handleFor($orderMessageId, $form);
+
+            if ($result->getIdentifiableObjectId()) {
+                $this->addFlash('success', $this->trans('Successful update.', 'Admin.Notifications.Success'));
+
+                return $this->redirectToRoute('admin_order_messages_index');
+            }
+        } catch (OrderMessageNotFoundException $e) {
+            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages()));
+
+            return $this->redirectToRoute('admin_order_messages_index');
+        } catch (Exception $e) {
+            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages()));
+        }
+
+        return $this->render('@PrestaShop/Admin/Sell/CustomerService/OrderMessage/edit.html.twig', [
+            'help_link' => $this->generateSidebarLink($request->attributes->get('_legacy_controller')),
+            'enableSidebar' => true,
+            'layoutTitle' => sprintf($this->trans('Edit: %s', 'Admin.Actions'), $orderMessageName),
```

Comment:
It is already localized in _Admin.Catalog.Feature_, but _Admin.Actions_ makes more sense in my opinion.

Reply:
:+1: